app/models/system_user.rb

```diff
@@ -0,0 +1,44 @@
+class SystemUser < ApplicationRecord
```

Comment:
We can change the class name for something like ```ReferralSystemEmailUser```

Reply:
Done!